### PR TITLE
fix: adds px to shadows

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Bug
- 
#### What is expected?

px into shadows

### CODE REVIEW

**Comment by chambo-e** [comment]
These changes have been made through scripts/figma-synchronise-tokens.sh ?

**Review by matthprost — CHANGES_REQUESTED** [review]
As @chambo-e said, it sould be done into the script where we use JQ, here you're editing a file that shouldn't be edited manually (its actually written at the top of the file in comment haha)

File: src/theme/tokens/dark.ts [hunk 1]
```diff
@@ -216,20 +216,20 @@ export default {
     },
   },
   shadows: {
-    bulk: '0 4 32 8 #00000066',
```
[the remark this answers is not in the document]
Comment (matthprost) [reply src/theme/tokens/dark.ts:219]: Cf. my message up ⬆️

**Comment by vincentaudebert** [comment]
@matthprost oopsie 😂

**Pull request closed without merging**